src/pages/Home/Home.tsx

```diff
@@ -179,6 +182,22 @@ const Home = () => {
             </h1>
           </Button>
         </div>
+        <h6 className={'font-semibold text-md'}>Legenda</h6>
+        <div className="flex flex-row gap-2 flex-wrap">
+          {Object.values(SupplyPriority)
+            .filter((value): value is number => typeof value === 'number')
+            .map((p, idx) => {
+              const { label, className } = getSupplyPriorityProps(p);
+              return (
+                <Chip
+                  key={idx}
```

Comment:
Como pode ter outros elementos usando esse mesmo padrão de key na página, sugiro mudar para algo mais único como por exemplo:
```suggestion
                  key={`subtitle-item-${idx}`}
```

Reply:
Parece bom pra mim